lib/components/primitive-components/Hole.ts

```diff
@@ -48,6 +49,10 @@ export class Hole extends PrimitiveComponent<typeof holeProps> {
     const { _parsedProps: props } = this
     const subcircuit = this.getSubcircuit()
     const position = this._getGlobalPcbPositionBeforeLayout()
+    const soldermaskMargin = props.solderMaskMargin
+      ? distance.parse(props.solderMaskMargin)
```

Comment:
yea you shouldn't need to call distance.parse, props must not have put in `distance` properly, this is already parsed with `_parsedProps`